packages/devops/scripts/deployment/deployPackages.sh

```diff
@@ -17,7 +17,7 @@ else
     ENVIRONMENT="dev"
 fi
 
-PACKAGES=("meditrak-server" "web-config-server" "psss-server" "lesmis-server" "report-server" "entity-server" "web-frontend" "admin-panel" "psss")
+PACKAGES=("meditrak-server" "web-config-server" "psss-server" "lesmis-server" "report-server" "server-boilerplate" "entity-server" "web-frontend" "admin-panel" "psss")
```

Comment:
Same here, I believe we don't need to built it if it's just an internal dependency